tests/e2e/cucumber/steps/ui/public.ts

```diff
@@ -169,3 +169,12 @@ When(
     await processDelete(stepTable, pageObject, actionType)
   }
 )
+
+Then(
+  'for {string} file {string} should be selected',
+  async function (this: World, stepUser: string, fileName: string): Promise<void> {
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    const resourceObject = new objects.applicationFiles.Resource({ page })
+    await resourceObject.checkBoxShouldBeChecked({ fileName: fileName })
```

Comment:
Why do we have to be specific about checkbox? may be just rename a fucntion `fileShouldBeSelected()` ?